python_impl/benchmarks/synthetic/generate_benchmark.py

```diff
@@ -23,7 +23,11 @@
 }
 def generateAIEplaceBenchmark(bench_name, bench_num):
 
-    mult = max(1, bench_num) # size multiplier
+    row_factor = 1 #random.choice(range(1, 9, 1))
+    mult = row_factor * max(1, bench_num) # size multiplier
+    num_rows = 8 * row_factor
+    num_cols = random.choice(range(5, 16, 5))
+
```

Comment:
Might be better to try some of the AIE configurations that exist (e.g. 4x5, 8x50, etc.)